packages/plugin-webpack/plugin.js

```diff
@@ -28,8 +28,8 @@ module.exports = function plugin(config, args) {
   }
   // Validate: args.outputPattern
   args.outputPattern = args.outputPattern || {};
-  const jsOutputPattern = args.outputPattern.js || "js/bundle-[hash].js";
-  const cssOutputPattern = args.outputPattern.css || "css/style-[hash].css";
+  const jsOutputPattern = args.outputPattern.js || "js/[name].[hash].js";
+  const cssOutputPattern = args.outputPattern.css || "css/[name].[hash].css";
```

Comment:
👍 Good improvement here

Reply:
🤔 should we use `[contenthash]` here instead of `[hash]` (hash based on file contents)? If you don’t have an opinion, we can leave it for now.

Reply:
Agree.

`[contenthash]` should be better than `[hash]`. I leave it unmodified just because it use `[hash]` before.

I will make an update.